jwst-visit-parser/build.sh

```diff
@@ -0,0 +1,2 @@
+
+$PYTHON setup.py install
```

Comment:
```suggestion
$PYTHON setup.py install --single-version-externally-managed --record record.txt
```

Reply:
Thank you @obviousrebel , this works for me locally and I am able to creare a new environment and import visitparser within that environment.